docs/rc.xml.all

```diff
@@ -25,6 +25,12 @@
       <slant>normal</slant>
       <weight>normal</weight>
     </font>
+    <font place="InactiveWindow">
+      <name>sans</name>
+      <size>10</size>
+      <slant>italic</slant>
+      <weight>normal</weight>
+    </font>
```

Comment:
Suggest reverting `slant` to `normal` here to a) match the hardcoded default in `src/config/rcxml.c` and b) to not surprise users by the sudden change in defaults.

Reply:
Done

Reply:
Fixed